var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -396,6 +396,14 @@ def libs(self):
     # https://www.intel.com/content/www/us/en/developer/articles/technical/building-boost-with-oneapi.html
     patch("intel-oneapi-linux-jam.patch", when="@1.76: %oneapi")
 
+    # https://github.com/boostorg/phoenix/issues/111
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/boostorg/phoenix/pull/112.patch",
```

Comment:
Can you use a stable url? like a commit sha, not a pull request number

Reply:
What? How is this not stable? It's definitely not easily possible to delete pull requests on github. In what sense is a commit sha from a contributors' fork better?

Reply:
@haampie Thanks for the suggestion, done